tekton/ci/catalog/template.yaml

```diff
@@ -35,6 +37,8 @@ spec:
         generateName: pull-catalog-catlin-run-
         labels:
           prow.k8s.io/build-id: $(tt.params.buildUUID)
+          tekton.dev/source-event-id: $(tt.params.sourceEventId)
+          tekton.dev/source-event-id: $(tt.params.sourceEventId)
```

Comment:
Duplicate line here.

Reply:
In fairness, it was `plumbing-yamllint` that caught it and I just pointed it out. ;)